core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -312,8 +312,8 @@ def validate_funding_spent(workbook: dict[str, pd.DataFrame]) -> list["TownsFund
     :return: ValidationErrors
     """
     # pull programme and project indexes from the workbook
-    programme_id = workbook["Programme_Ref"]["Programme ID"][0]
-    fund_type = workbook["Programme_Ref"]["FundType_ID"][0]
+    programme_id = workbook["Programme_Ref"].iloc[0]["Programme ID"]
+    fund_type = workbook["Programme_Ref"].iloc[0]["FundType_ID"]
```

Comment:
Has this been included in a seperate PR already?

Reply:
Yeah this is in another PR. Just had to rebase this and it's gone from the changes now 👍